ios/brave-ios/Sources/Brave/BraveSkus/BraveSkusManager.swift

```diff
@@ -104,14 +104,16 @@ public class BraveSkusManager {
         
         switch credentialSummaryJson.state {
         case .valid:
-          if Preferences.VPN.skusCredential.value == nil {
-            Logger.module.debug("The credential does NOT exists, calling prepareCredentialsPresentation")
-            self.prepareCredentialsPresentation(for: domain, path: "*") { _ in
-              // Keep the skus manager alive until preparing credential presentation finishes.
-              _ = self
+          if domain.hasPrefix("vpn.brave") {
```

Comment:
It can't. This script is only ever injected on account.brave*.com

The domain has to be vpn.brave*.com or leo.brave*.com otherwise the credentials will fail.

This check is just to see which credentials the website wants :)